### PR TITLE
Add importing section to menu

### LINKED ISSUE #587: [DOCS] Add FAQ section / cjs/umd section
1. useField has to be used in form

- a lot of people encounters this problem and they don't how to deal with that

### PR DESCRIPTION
**Pull request by rvsia**
closes #587

### CODE REVIEW

File: packages/react-renderer-demo/src/pages/importing.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+Data Driven Forms is built in three module formats: UMD, CJS and ESM.
+
+## UMD
+
+```jsx
+import FormRenderer, { useField } from '@data-driven-forms/react-form-renderer';
+```
+
+Only the `FormRenderer` component is imported as a default. In all other packages, use only named exports.
```
Comment (Hyperkid123) [inline packages/react-renderer-demo/src/pages/importing.md:15]: this should say exported

File: packages/react-renderer-demo/src/pages/importing.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+import DocPage from '@docs/doc-page';
+
+<DocPage>
+
+# Importing
+
+Data Driven Forms is built in three module formats: UMD, CJS and ESM.
+
+## UMD
```
Comment (Hyperkid123) [inline packages/react-renderer-demo/src/pages/importing.md:9]: I would definitely stress that UMD is there just as a fallback option and should be used if you are building inside CJS or ESM env.

**Comment by rvsia** [comment]
I think calling using of different import as a optimization would be a little bit exaggerated.  :smile_cat:

**Comment by Hyperkid123** [comment]
Yes. It is optimization. The only reason why different builds exist in this project is bundle size optimization. There is no reason to use anything other builds than UMD if you ignore the optimization. For mappers at least. You can reduce the bundle size by a lot by not using components like wizard, select, or date pickers. This is a subject of optimizing applications.

And adding the babel transform example is a great example of how you can automate it. We all sometimes forget to use the correct import. Especially if you are another library (FCE) with multiple builds.

**Pull request merged**